stdlib/3/os/__init__.pyi

```diff
@@ -425,7 +425,7 @@ else:
 
 if sys.version_info >= (3, 3):
     @overload
-    def listdir(path: Optional[str] = ...) -> List[str]: ...
+    def listdir(path: Optional[_PathType] = ...) -> List[str]: ...
```

Comment:
`_PathType` includes bytes, so I'm surprised mypy is OK with this (shouldn't the overloads overlap?). We should make sure that mypy still infers `os.listdir(b'.')` as returning `List[bytes]`.